modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -481,7 +488,32 @@ using GConstGIEModel = ade::ConstTypedGraph
     , IECallable
     >;
 
+inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i) {
+    GAPI_Assert(ctx.inShape(i) == cv::GShape::GFRAME &&
+                        "Callback feature is supported for MediaFrame only");
+
+    cv::util::any any_blob_params = ctx.inFrame(i).blobParams();
+    auto ie_core = cv::gimpl::ie::wrap::getCore();
+
+    using ParamType = std::pair<InferenceEngine::TensorDesc,
+                                InferenceEngine::ParamMap>;
+
+    ParamType* blob_params = cv::util::any_cast<ParamType>(&any_blob_params);
+    if (blob_params == nullptr) {
+        GAPI_Assert(false && "Incorrect type of blobParams: "
+                              "expected std::pair<InferenceEngine::TensorDesc,"
+                                                 "InferenceEngine::ParamMap>");
```

Comment:
This shouldn't be an issue. If a remote blob can't be created, create it normally as it is done today (obtain a host-side pointer from `MediaFrame::View` and so on).

Reply:
Answered in another comment

Reply:
Which one?

Reply:
Please post a link or quote here as I don't see it in the review.

Reply:
Discussed locally. Lets solve this problem separately

Reply:
Agree.